submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/K8sSubmitter.java

```diff
@@ -254,6 +267,19 @@ public Notebook createNotebook(NotebookSpec spec) throws SubmarineRuntimeExcepti
       Map<String, String> labels = new HashMap<>();
       labels.put(NotebookCR.NOTEBOOK_OWNER_SELECTOR_KET, spec.getMeta().getOwnerId());
       notebookCR.getMetadata().setLabels(labels);
+
+      // create persistent volume
+      createPersistentVolume(notebookCR.getMetadata().getName());
+
+      // create persistent volume claim
+      createPersistentVolumeClaim(notebookCR.getMetadata().getNamespace(),
+              notebookCR.getMetadata().getName());
+
+      // bind persistent volume claim
+      V1PersistentVolumeClaimVolumeSource pvc_source = new V1PersistentVolumeClaimVolumeSource()
+              .claimName(notebookCR.getMetadata().getName() + "-claim");
```

Comment:
Should generate the claim name use the same method, so that avoid future mistakes.